VatSpace/cats.lua

```diff
@@ -58,6 +58,7 @@ local function createCatInSpace()
 end
 
 function M.move()
+    ui.staticElementsToFront()
```

Comment:
This function run 60 times per second, are you sure that you want use it here? Try find another place.

Reply:
My bad, sorry :) Fixed.